app/src/views/ForgotPasswordCode.tsx

```diff
@@ -26,6 +32,18 @@ export const ForgotPasswordCode = () => {
     setFieldValue('code', code);
   };
 
+  if (!email) {
+    return (
+      <FormContainer>
```

Comment:
In a future changeset we may be able to factor this out so that it is reused here and in the `ResetPassword` component